doc/tech/release_notes/baldy.md

```diff
@@ -0,0 +1,121 @@
+[<< Back](../)
+# Baldy Release Notes
+
+
+## Baldy Release Contents
+* [Overview](#overview)
+* [RM (v3.0)](#rm)
+* [RA-1 (v2.0)](#ra1)
+* [RI-1 (v1.0)](#ri1)
+* [RC-1 (v1.0-pre-alpha)](#rc1)
+* [RA-2 (v1.0)](#ra2)
+
+<a name="overview"></a>
+## Overview
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.tech.1` 	| VNF Revolution policy and strategy 	|  	|
+| `baldy.tech.2` 	| Future Roadmap |  |
+| `baldy.tech.3` 	| TBD |  |
+
+*Baldy Release should include at least features up to and including `baldy.tech.2`.
+
+<a name="rm"></a>
+## Reference Model (v3.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.rm.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.rm.2` 	| Limiting infrastructure profiles to Basic and Network Intensive (Parking Compute intensive) | Ch02, Ch04, and Ch05 |
+| `baldy.rm.3` 	| Finalising Compliance, Verification and Conformance Strategy | Ch08 	|
+| `baldy.rm.4*` 	| Full Container support 	| Ch04 	|
+| `baldy.rm.5` 	| Complete Security Chapter | Ch07: 100% alignment with ONAP 	|
+| `baldy.rm.6` 	| Virtual Networking/ Networking Fabric Modelling 	| Ch03 	|
+| `baldy.rm.7` 	| Generic Installer Model 	| Ch09 	|
+| `baldy.rm.8` 	| Guidelines 	| Appnedix-A 	|
+
+*Baldy Release should include at least features up to and including `baldy.rm.4`.
+
+<a name="ra1"></a>
+## Reference Architecture 1 (v2.0)
+
+>WSL Note: Features below should be implemented in order.
+
+| Reference # 	| Feature 	| Notes 	|
+|-------------	|-----------------	|-------	|
+| `baldy.ra1.1` 	| General Cleanup 	| All Chapters 	|
+| `baldy.ra1.2` 	| Clarify OpenStack version policy 	| Ch01, Ch05 	|
+| `baldy.ra1.3` 	| Align Requirement to RM requirements 	| Ch02 	|
```

Comment:
BTW There is no single RM chapter on requirements. We are currently developing RM Security CH07 requirements and engaged in cleaning up the requirements listed in RM Ch04 and 05.

This should be relabeled as incorporate RM Requirements in RA-1 Requirements

Reply:
OK understood. done.